includes/admin.form.inc

```diff
@@ -40,6 +40,17 @@ function islandora_manuscript_admin_settings_form(array $form, array &$form_stat
           array('pdf', 'image', 'ocr'))),
       ),
     ),
+    // Metadata display.
+    'metadata_display_fieldset' => array(
+      '#type' => 'fieldset',
+      '#title' => t('Metadata display'),
+      'islandora_collection_metadata_display' => array(
+        '#type' => 'checkbox',
+        '#title' => t('Display object metadata'),
+        '#description' => t('Display object metadata below the collection display.'),
+        '#default_value' => variable_get('islandora_collection_metadata_display', FALSE),
```

Comment:
This should be added to the uninstall hook https://github.com/discoverygarden/islandora_solution_pack_manuscript/blob/7.x/islandora_manuscript.install#L21-Lundefined